### PR TITLE
Changes in MySQL plugin

### PR DESCRIPTION
**Pull request by paulfantom**
PR to issue #520

### CODE REVIEW

File: charts.d/mysql.chart.sh [hunk 1]
```diff
@@ -10,136 +10,26 @@
...
-	local ret
-
-	"${@}" -s -e "show global status;"
-	ret=$?
+mysql_get() {
+	local arr
+	local oIFS="${IFS}"
+	mysql_data=()
+	IFS=$'\t'$'\n'
+	arr=($("${@}" -e "SHOW GLOBAL STATUS WHERE value REGEXP '^[0-9]';" | egrep "^(Bytes|Slow_|Queri|Handl|Table|Selec|Sort_|Creat|Conne|Abort|Binlo|Threa|Innod|Qcach|Key_|Open)" ))
```
Comment (ktsaou) [inline charts.d/mysql.chart.sh]: we don't need `-s` too ?

Comment (paulfantom) [inline charts.d/mysql.chart.sh]: It runs with and without `-s` and as I recall there was minor time penalty when running mysql with it.

**Comment by ktsaou** [comment]
ok. merge it?

**Comment by paulfantom** [comment]
I think so.

**Comment by ktsaou** [comment]
thanks!

**Pull request merged**